Resources/Styles/package_list.scss

```
@@ -35,10 +35,22 @@
         margin: 5px 0;
         color: var(--page-text);
       }
+
     }
 
     &.selected {
       background-color: var(--package-list-selectedbackground);
     }
   }
+
+  #star_text {
+    background-image: var(--image-star);
+    background-repeat: no-repeat;
+    background-size: 30%;
+    background-position: center;
+    padding: 25px;
+  }
```

Updated to match the suggestion to use a `class` instead of an `id`. I also made the size of the image explicit rather than percentage-based, in case the size of the `p` ever changes.

```suggestion
  .stars {
    padding-left: 25px;
    background-size: 18px;
    background-repeat: no-repeat;
    background-position: top 2px left;
    background-image: var(--image-star);
  }
```